src/plugins/i18n/locales/en.json

```diff
@@ -276,6 +281,25 @@
   "imageView": {
     "dialogDesc": "Downloading image..."
   },
+  "sortOptions": {
+    "Top": "Top",
+    "top": {
+      "top": "Hour",
+      "Hour": "Six Hours",
+      "SixHours": "Twelve Hours",
+      "TwelveHours": "Day",
+      "Day": "Week",
+      "Week": "Month",
+      "Month": "Year",
+      "Year": "All Time"
+    },
+    "AllTime": "Hot",
+    "Hot": "Active",
+    "Active": "New",
+    "New": "Most Comments",
+    "MostComments": "New Comments",
+    "NewComments": "Old"
+  },
```

Comment:
These are all wrong and don't match the `de.json` options either.

Reply:
Thanks for the warning, my bad

Reply:
Np